frontend/src/features/projects/common/interfaces.ts

```diff
@@ -125,31 +125,38 @@ export interface IProject {
   publicNote: string;
   privateNote: string;
   agencyResponseNote?: string;
+  offersNote?: string;
   agencyId: number;
   statusId: number;
   status?: IStatus;
   exemptionRationale?: string;
   exemptionRequested?: boolean;
-  statusCode: string;
+  statusCode?: string;
   tierLevelId: number;
   tasks: IProjectTask[];
   rowVersion?: string;
   confirmation?: boolean;
-  approvedOn?: Date;
-  deniedOn?: Date;
-  cancelledOn?: Date;
-  submittedOn?: Date;
-  initialNotificationSentOn?: Date;
-  thirtyDayNotificationSentOn?: Date;
-  sixtyDayNoficationSentOn?: Date;
-  ninetyDayNotificationSentOn?: Date;
-  onHoldNotificationSentOn?: Date;
-  transferredWithinGreOn?: Date;
-  clearanceNotificationSentOn?: Date;
+  approvedOn?: Date | string;
+  deniedOn?: Date | string;
+  cancelledOn?: Date | string;
+  submittedOn?: Date | string;
+  initialNotificationSentOn?: Date | string;
+  thirtyDayNotificationSentOn?: Date | string;
+  sixtyDayNoficationSentOn?: Date | string;
+  ninetyDayNotificationSentOn?: Date | string;
+  onHoldNotificationSentOn?: Date | string;
+  transferredWithinGreOn?: Date | string;
+  clearanceNotificationSentOn?: Date | string;
+  marketedOn?: Date | string;
+  disposedOn?: Date | string;
+  offerAcceptedOn?: Date | string;
   netBook?: number;
   assessed?: number;
   estimated?: number;
-  workflowCode?: string;
+  workflowCode?: string | '';
+  offerAmount?: number | '';
+  isContractConditional?: boolean;
+  purchaser?: string | '';
```

Comment:
This is all to do with formik not allowing controlled fields to have the value of undefined.